### PR TITLE
Fixes #13

### LINKED ISSUE #13: Export to JSON

### PR DESCRIPTION
**Pull request by jatinAroraGit**
Hey there,
I added a function named create_json in popup.js to export parameters as JSON file. 
Since there was no mention of implementing any new button or UI for supporting this new feature, I just made the function for fixing this issue.
Let me know if there are any problems with the PR.
Have a good day.

### CODE REVIEW

**Comment by inuyasha82** [comment]
Hi Jatin, i will have a look at the pull request during the weekend (probably on sunday) btw. Oh yeah i forgot to create a new issue for the UI component. That has to be updated somehow. I'll create one.

File: src/popup.js [hunk 1]
```diff
@@ -270,3 +270,12 @@ function create_csv(parameters_array){
     }
     return csv_file;
 }
+function create_json(parameter_array){
```
Comment (inuyasha82) [inline src/popup.js:279]: minor thing: add an empty line between the two functions, and also add a comment similar to the export_csv

**Review by inuyasha82 — CHANGES_REQUESTED** [review]
Another thing you can do is to add the json exporter to the list of exporters (even if not ready yet, it can be added to the exporters variable (check line 9), you should just need to add something like: 

json: create_json

File: src/popup.js (hunk 1 above)
Comment (jatinAroraGit) [reply src/popup.js:279]: Hey @inuyasha82 
Did a new commit https://github.com/inuyasha82/chrome_parameters/pull/67/commits/41a25511c265c90d5015528896415a87803bd0d2 addressing the comments.

Comment (inuyasha82) [reply src/popup.js:279]: thank you!

File: src/popup.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ function click(e) {
 		console.log("Add new parameter");
 		add_new_parameter();
 	} else if(e.target.id=="export"){
-		export_parameters_list("csv");
+		export_parameters_list("json");
```
Comment (inuyasha82) [inline src/popup.js]: Hey @jatinAroraGit can you revert this to csv please? it will be fixed on the next PR

Comment (jatinAroraGit) [reply src/popup.js]: Sure @inuyasha82 .
Did it in 4b763b0

Comment (inuyasha82) [reply src/popup.js]: thanks

**Review by inuyasha82 — APPROVED** [review]

**Comment by inuyasha82** [comment]
Merged, thank you!

**Pull request merged**